services/brig/src/Brig/API.hs

```diff
@@ -1133,30 +1157,33 @@ getUserNameH (_ ::: self) = do
     Just n -> json $ object ["name" .= n]
     Nothing -> setStatus status404 empty
 
-listUsersH :: JSON ::: UserId ::: Either (List UserId) (List Handle) -> Handler Response
+listUsersH :: JSON ::: UserId ::: Either (List OpaqueUserId) (List Handle) -> Handler Response
```

Comment:
Should the handles be qualified here?

Reply:
Optionally qualified, right? Good catch.

I don't think we are using this anywhere right now (we are definitely not in web) and I'm not sure what the use case is, but I'll fix it.

Reply:
done